onebusaway-android/src/amazon/java/org/onebusaway/android/map/googlemapsv2/VehicleOverlay.java

```diff
@@ -91,25 +92,30 @@
 
     private static final int NORTH = 0;
 
-    private static final int NORTH_WEST = 1;
```

Comment:
I'll test more myself too.  I loaded it up and at a glance icons looked correct based on route directions.